### ISSUE TITLE
runSymfonyConsoleCommand with options is sometimes ignoring the options

### ISSUE THREAD

**Issue by MarkDaleman** [Codeception/module-symfony#172]
Hello 👋 

I'm facing a problem when loading migrations from my Module. I want to hook in on the `_beforeSuite` with a Symfony command to load my fixtures and such.

The interesting thing is, one part works fine while the other part doesn't.

I'm trying to run the following code:

```php
$symfony->runSymfonyConsoleCommand('doctrine:fixtures:load', ['--no-interaction'=>true])
```

When I print this I get the following rest:

```bash
 Careful, database "db_name_here_test" will be purged. Do you want to continue? (yes/no) [no]:\n
 > 
```

I also run other commands like:

```php
$symfony->runSymfonyConsoleCommand('doctrine:database:drop',['--if-exists'=>true, '--force'=>true]);
$symfony->runSymfonyConsoleCommand('doctrine:database:create');
$symfony->runSymfonyConsoleCommand('doctrine:migrations:migrate', ['--no-interaction'=>true]);
```

And those work fine (and without user interaction).

So the question is, do I something very wrong or is there a possible bug?

**More information**

What I've tried is:

- replacing --no-interaction with --quiet

What kind of software I'm using:

- Symfony 6.4
- CodeCeption 5.0.12
- Module-symfony 3.11

My module's full code for reference:

```php
<?php

namespace App\Tests\Helper;

use Codeception\Exception\ModuleException;
use Codeception\Module;

class MyModule extends Module
{
    /**
     * @throws ModuleException
     */
    public function _beforeSuite(array $settings = []): void
    {
        /** @var Module\Symfony  $symfony */
        $symfony = $this->getModule('Symfony');
        $symfony->runSymfonyConsoleCommand('doctrine:database:drop',['--if-exists'=>true, '--force'=>true]);
        $symfony->runSymfonyConsoleCommand('doctrine:database:create');
        $symfony->runSymfonyConsoleCommand('doctrine:migrations:migrate', ['--no-interaction'=>true]);
        $symfony->runSymfonyConsoleCommand('doctrine:fixtures:load', ['--no-interaction'=>true]);
    }
}
```

**Comment by mkilmanas** [comment]
To confirm and clarify the issue here: the problem exists only for some key options that are handled in the special way

E.g. `--no-interaction` should make `$input->isInteractive() === false`, similarly none of the `-v, -vv, -vvv, --verbose` options work as they would also be setting the verbosity level on the Output object.

In particular:
- these options are passed as `$parameters` to [ConsoleAssertionsTrait::runSymfonyConsoleCommand()](https://github.com/Codeception/module-symfony/blob/19c86fc/src/Codeception/Module/Symfony/ConsoleAssertionsTrait.php#L28)
- which in turn uses them as the only (first) parameter `$input` when calling [CommandTester::execute()](https://github.com/symfony/console/blob/v6.4.1/Tester/CommandTester.php#L49)
- But CommandTester actually uses only the second parameter [`$options` when initializing Output object](https://github.com/symfony/console/blob/v6.4.1/Tester/CommandTester.php#L72)
- Furthermore, [TesterTrait::initOutput()](https://github.com/symfony/console/blob/v6.4.1/Tester/TesterTrait.php#L139-L141) actually expects the verbosity option as a parsed integer level - so I suppose it has to be pre-parsed already
- As for 'interactive' mode, it also is supposed to come via `$options` and not in the `$input` to [CommandTester::execute()](https://github.com/symfony/console/blob/v6.4.1/Tester/CommandTester.php#L64-L66)

**Comment by TavoNiievez** [comment]
Hi @MarkDaleman ,

I have opened a Pull Request https://github.com/Codeception/module-symfony/pull/188 to resolve this issue.
with it, a test like the following now works correctly:
```php
        $output = $I->runSymfonyConsoleCommand('doctrine:fixtures:load', ['-q']);
        $I->assertIsEmpty($output);
```
@mkilmanas ' explanation was quite helpful in writing the code.

For me it was important to make this change without changing the signature of the `runSymfonyConsoleCommand` method, this way I avoid creating BC and having to release a new major version just for this.

This manual parsing is not the nicest solution, and I guess Symfony itself must do something similar in its own code, but for the purposes of solving this issue it works quite well.

@mkilmanas If something inhibited you from submitting a PR yourself for this having such a clear understanding of the problem please let me know, likewise if you can do a little Code Review to the PR I'd appreciate it.

Thanks a lot to both of you!

**Comment by mkilmanas** [comment]
@TavoNiievez thanks for the PR - looks good to me!

I did not get involved with this as figuring out the cause of the problem is one thing and providing a solution that would be in line with the rest of the module is another - I am not that familiar with the internals of codeception or its modules.

**Comment by TavoNiievez** [comment]
Released as [3.3.2](https://github.com/Codeception/module-symfony/releases/tag/3.3.2)

Thank you!
